src/rotten-tomatoes/index.js

```diff
@@ -1,3 +1,72 @@
-function rottenTomatoes(grid) {}
+function rottenTomatoes(grid) {
+  let tempHolder = []
+  let rotLevel = []
+  let minutes = 0
+  //gets all good tomatoes adjacent to rotten tomatoes
+  const getAdjacent1 = (index) => {
+    let row = index[0]
+    let column = index[1]
+    const above = grid[row - 1] === undefined ? undefined : grid[row - 1][column]
+    const below = grid[row + 1] === undefined ? undefined : grid[row + 1][column]
+    const right = grid[row][column + 1]
+    const left = grid[row][column - 1]
+    if(above === 1) {
+      tempHolder.push([row - 1, column])
+      grid[row - 1][column] = 0
+    }
+    if(below === 1) {
+      tempHolder.push([row + 1, column])
+      grid[row + 1][column] = 0
+    }
+    if(right === 1) {
+      tempHolder.push([row, column + 1])
+      grid[row][column + 1] = 0
+    }
+    if(left === 1) {
+      tempHolder.push([row, column - 1])
+      grid[row][column - 1] = 0
+    }
+
+  }
+  //function to expand the level of rot
+  const expandRot = (level) => {
+    level.forEach(element => {
+      getAdjacent1(element)
+    });
+    rotLevel = [...tempHolder]
+    tempHolder = []
+  }
+//fucntion to search for items in the grid
+  const find = num => {
+    for (let i = 0; i < grid.length; i += 1) {
+      for (let j = 0; j < grid[i].length; j += 1) {
+        if (grid[i][j] === num) {
+          rotLevel.push([i, j])
+          grid[i][j] = 0
+        }
+      }
+    }
+  }
+
+//find all the sources of the rot
+  find(2)
```

Comment:
This is confusing. It doesn't return anything and just mutates things. You should prolly extract all these helper functions out.